modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * installing/installing_gcp/installing-gcp-account.adoc
 
+:_mod-docs-content-type: CONCEPT
```

Comment:
This module doesn't really fit into any of the mod docs types as it's written, IMO. Since it's saying that the reader must do something in the course of a task flow and has that "you can skip this section part," it resembles a procedure, though it's not in that format.

https://redhat-documentation.github.io/modular-docs/#_creating_modules

Reply:
Note - while this module doesn't quite fit into any of the mod doc types, we will look to restructure it in a future update. For now the module will be categorized as a procedure.